index.html

```diff
@@ -10,951 +12,1033 @@
 </head>
 
 <body>
-  <div class="hero hero-bg is-fullheight">
+  <header class="hero hero-bg is-fullheight">
     <div class="hero-head">
       <div class="navbar">
         <div class="container">
           <div class="navbar-brand">
             <a class="navbar-item logo-link">
-              <img src="https://www.epam.com/content/dam/epam/global/logo_white-blue.svg">
+              <img alt="EPAM Systems logotype" src="https://www.epam.com/content/dam/epam/global/logo_white-blue.svg">
             </a>
             <span class="navbar-burger burger" data-target="navbarMenu">
```

Comment:
I found JS code in the `js/custom.js` for the burger but I don't see the layout.
If you want to implement logic on the UI for the burger using tab, please add the following code:

```suggestion
 <span class="navbar-burger burger" data-target="navbarMenu">
    <button class='burger-hidden' type="button" tabindex="0">Click here to open the menu</button>
   <nav id="navigation">
	 <span></span>
	 <span></span>
	 <span></span>
  </nav>
  </span>
  
  
  .burger-hidden {
  border: 0;
  width: 50px;
  height: 50px;
  overflow: hidden;
  color: transparent;
  background-color: transparent;
}

.burger-hidden:focus {
  outline: 0;
  border-radius: 2px;
  box-shadow: 0 0 2px 1.5px #bc6e6e, 0 0 3px 3px #1274C4;
}
```

Please fix the queue of the focuses using `tabindex` attr

Reply:
burger menu is from the original project, I wanted to do something different but I didn't add my code